<?xml version="1.0" encoding="UTF-8"?>
<launch>
<arg name="pick_yaml"  default="pick_on_table_simple_manuale"/>
  <arg name="place_yaml"  default="prima_prova_task"/>
  <arg name="tiago_yaml"  default="tiago_poses_2"/>
  

  <rosparam command="load" ns="pick_task"  file="$(find base_placement_plugin)/saved_tasks/$(arg pick_yaml).yaml" />
    <rosparam command="load" ns="place_task"  file="$(find base_placement_plugin)/saved_tasks/$(arg place_yaml).yaml" />
    <rosparam command="load" ns="tiago_task"  file="$(find tiago_pick_and_place)/config/$(arg tiago_yaml).yaml" />

</launch>
